src/webgl/vertex-array.js

```diff
@@ -165,12 +166,9 @@ export default class VertexArray extends Resource {
 
   // Disable an attribute
   disable(location) {
-    // Don't disable location 0
```

Comment:
Nit: I see... Seems to me like this obscure condition still somehow does belong in disable(). Maybe add a flag to disable(.., {disableZero = false} = {}) instead?